projects/plugins/jetpack/modules/seo-tools/class-jetpack-seo-titles.php

```diff
@@ -93,17 +93,28 @@ public static function get_custom_title( $default_title = '' ) {
 			return $default_title;
 		}
 
+		if ( ! Jetpack_SEO_Utils::is_enabled_jetpack_seo() ) {
+			return $default_title;
+		}
+
+		// If it's a singular -- page or post -- check for a meta title override.
+		if ( 'pages' === $page_type || 'posts' === $page_type ) {
+			$post = get_post();
+			if ( $post instanceof WP_Post ) {
+				$custom_title = get_post_meta( $post->ID, Jetpack_SEO_Posts::HTML_TITLE_META_KEY, true );
+				if ( $custom_title ) {
```

Comment:
Thoughts on checking `! empty()` here to catch a title of only whitespace?

Reply:
Is there a (valid) use case for only a whitespace title? I can't think of one. In addition to checking when pulling it, check when saving it and delete the meta when it is only whitespace?

Reply:
I believe we could check for the whitespaces case here, but indeed, more importantly, we shouldn't save such a string first of all, and it should be handled on the frontend.

Reply:
Good points - agreed, we do not have a use case for an empty title. 

We could prevent an empty string from being entered on the front end and potentially utilize `update_post_metadata` and `add_post_metadata` filters to check and prevent an empty string from being saved on the back end.

Reply:
Setting empty titles is currently blocked on the frontend, and a check before displaying was added in 25c949f